robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -127,7 +127,7 @@ def get_closest_room(self, center_point=None, radius=0):
 
         return self.get_closest_entity(type="room", center_point=center_point, radius=radius)
 
-    def get_closest_laser_entity(self, type="", center_point=kdl.Vector(), radius=0):
+    def get_closest_laser_entity(self, type="", center_point=VectorStamped(), radius=0):
         if isinstance(center_point, PointStamped):
```

Comment:
No callers pass center_point=PointStamped, only VectorStamped. So check is removed in 2f780fd

Transformation to /map happens in get_entity that is called internally.